cypress-tests/package.json

```diff
@@ -9,5 +9,8 @@
     "cypress:ci": "npx cypress run --headless"
   },
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "jsqr": "^1.4.0"
```

Comment:
This dependency has been added intentionally and is used in [line 770 of commands.js](https://github.com/juspay/hyperswitch/blob/b5e27fdc39954ac785ee29977844e8f3d9c6d54f/cypress-tests/cypress/support/commands.js#L770) (`fetchAndParseQRCode`) which is called in [line 881 ](https://github.com/juspay/hyperswitch/blob/b5e27fdc39954ac785ee29977844e8f3d9c6d54f/cypress-tests/cypress/support/commands.js#L881)just so that we can parse the QR the code and read the value.